src/utils/profile.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+import _ from 'lodash';
+import type { Props } from '../screens/ProfileScreen/EditProfile';
+
+/**
+ * @desc Function that checks for the save enabled status
+ * @param {Props} props props that are passed to the component
+ * @returns {boolean} Save enabled status
+ */
+export default function saveShouldBeEnabled(props: Props): boolean {
```

Comment:
Utils should not depend on specific component Props type, since utils is used by components but not vice versa, so dependency should go the same way. However, it's a good idea to extract such function, so I suggest to change function so it takes `user` and `editingUser`.